.github/workflows/create_release.yml

```
@@ -0,0 +1,68 @@
+name: Create release
+
+on: 
+  workflow_dispatch:
+    
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [20.x]
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GIT_TOKEN }}
+
+      - name: Setup PNPM
+        uses: pnpm/action-setup@v2.4.0
+        with:
+          version: latest
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'pnpm'
+          cache-dependency-path: package.json
+
+      - name: Install standard-version and conventional-changelog
+        run : pnpm i -g standard-version conventional-changelog-cli
+
+      - name: Config git
+        run: |
+          git config --global user.email "${{ github.event.sender.email }}"
+          git config --global user.name "${{ github.event.sender.login }}"
+
+      - name: Create changelog
+        run: conventional-changelog -p conventionalcommits -r 0
+          
+      - name: Apply standard-version
+        run: |
+          standard-version -t ""  --no-verify --first-release
+
+      - name: Get new version
+        id: version
+        run: echo "VERSION=$(jq .version package.json -r)" >> $GITHUB_OUTPUT
+      
+      - name: Fix packages
+        run: |
+          sudo apt-get install moreutils
+          jq '.version = "${{ steps.version.outputs.VERSION }}"' packages/nestjs-firebase-admin/package.json | sponge packages/nestjs-firebase-admin/package.json
+          
+      - name: Commit release
+        run: |
+          git add .
+          git commit --amend --no-verify --no-edit
+          git push --force
+          git push --tags
```

Ohh I see :)

What if we do not ammend this commit and instead push a new commit:

```suggestion
      - name: Commit release
        run: |
          git add .
          git commit --no-verify -m "chore: bump version numbers"
          git push
          git push --tags
```

---

I am a fan of not amending commits, I love to maintain an explicit git history. But this case is a bit special. standard-version is increasing the version and creating a tag with the new version. We are adding new file changes that we want to include in the release that has already been tagged. Therefore, if we make a new commit, all the changes we just made will be outside of the release that has just been tagged.

---

Can we commit this change before appliying the tag? That would be the best

---

Changed :smile: